pages/docs/tutorials/native/basic-kotlin-native-app.md

```diff
@@ -2,90 +2,282 @@
 type: tutorial
 layout: tutorial
 title:  "A Basic Kotlin/Native Application"
-description: "A look at how to compile our first Kotlin/Native application"
-authors: Hadi Hariri 
-date: 2017-12-03
-showAuthorInfo: false
+description: "A look at how to compile our first Kotlin/Native application and open it in an IDE"
+authors: 
+  - Hadi Hariri
+  - Eugene Petrenko 
+date: 2019-04-15
 ---
 
 
-In this tutorial we'll see how to
+In this tutorial, we'll look at how to
 
 * [Obtain the Kotlin/Native compiler](#obtaining-the-compiler)
 * [Write the application](#creating-hello-kotlin)
-* [Compile and examine output](#compiling-and-examining-output)
+* [Create build files](#creating-a-kotlinnative-gradle-project)
+* [Set up an IDE](#open-in-ide)
 * [Run the application](#running-the-application)
 
 
-## Obtaining the compiler
+## Obtaining the Compiler
 
-Kotlin/Native is available for macOS, Linux and Windows. Depending on the operating system we're working on, we'll need to download
-the correct compiler. While cross-platform compilation is possible (i.e. using one platform to compile for another), in this first tutorial
-we're going to compile for the same operating system we're running on. In our case this will be macOS. 
+Kotlin/Native compiler is available for macOS, Linux, and Windows. It support
```

Comment:
The Kotlin/Native compiler is available for macOS, Linux, and Windows. It supports